.github/actions/build-binaries/action.yaml

```diff
@@ -40,22 +34,31 @@ runs:
 
     - name: Zip binaries
       shell: bash
-      continue-on-error: true
-      if: ${{ inputs.upload_binaries == 'true' }}
       run: |
         ls -l dist
         cd dist/algokit/
-        tar -zcvf ../../algokit-${{ env.RELEASE_VERSION }}-${{ inputs.operating_system }}-py${{ inputs.python_version }}.tar.gz *
+        if [ "${{ inputs.upload_binaries }}" == "true" ]; then
+          tar -zcvf ../../algokit-${{ env.RELEASE_VERSION }}-${{ inputs.operating_system }}-py${{ inputs.python_version }}.tar.gz *
+        else
+          tar -zcvf ../../algokit-${{ inputs.operating_system }}-py${{ inputs.python_version }}.tar.gz *
+        fi
         cd ../..
         ls -l
 
-    - name: Upload binary as artifact
+    - name: Upload binary as artifact (release)
       if: ${{ inputs.upload_binaries == 'true' }}
       uses: actions/upload-artifact@v4
       with:
-        name: algokit-cli-${{ inputs.operating_system }}-py${{ inputs.python_version }}
+        name: algokit-${{ inputs.operating_system }}-py${{ inputs.python_version }}
         path: algokit-${{ env.RELEASE_VERSION }}-${{ inputs.operating_system }}-py${{ inputs.python_version }}.tar.gz
 
+    - name: Upload binary as artifact (dev)
+      if: ${{ inputs.upload_binaries != 'true' }}
```

Comment:
@aorumbayev Should this input be renamed to something else?
Also should we be flagging these artefacts as pre-release?